README.md

```diff
@@ -196,18 +196,7 @@ use Zenstruck\Foundry\ModelFactory;
 use Zenstruck\Foundry\Proxy;
 
 /**
- * @method static Post|Proxy createOne(array $attributes = [])
- * @method static Post[]|Proxy[] createMany(int $number, $attributes = [])
- * @method static Post|Proxy find($criteria)
- * @method static Post|Proxy findOrCreate(array $attributes)
- * @method static Post|Proxy first(string $sortedField = 'id')
- * @method static Post|Proxy last(string $sortedField = 'id')
- * @method static Post|Proxy random(array $attributes = [])
- * @method static Post|Proxy randomOrCreate(array $attributes = []))
- * @method static Post[]|Proxy[] all()
- * @method static Post[]|Proxy[] findBy(array $attributes)
- * @method static Post[]|Proxy[] randomSet(int $number, array $attributes = []))
- * @method static Post[]|Proxy[] randomRange(int $min, int $max, array $attributes = []))
+ * @extends ModelFactory<\App\Entity\Post>
```

Comment:
The `Post` entity is imported above so the FQCN is not required here.

Have your confirmed auto-completion works as expected in the latest version of PHPStorm? I've had a lot of problems with generic support in this version.

If so, should we update the maker template?

Reply:
As discussed with @Nyholm in slack, I think we need to keep the existing annotations here and in the factory maker template. At least for the foreseeable future. I'd like to have auto-completion work for the widest spectrum of users as possible.